core/src/main/java/org/mskcc/cbio/oncokb/util/IndicatorUtils.java

```diff
@@ -346,6 +346,16 @@ public static IndicatorQueryResp processQuery(Query query, String geneStatus,
                     new ArrayList<>(relevantAlterations), query));
             }
 
+            // Prognostic summary
+            if (evidenceTypes.contains(EvidenceType.PROGNOSTIC_SUMMARY)) {
```

Comment:
Adding proper prognostic/diagnostic summaries need some work. Will be added in a separate PR.